src/main/java/com/epam/training/tasks/third/data/CubeDataValidator.java

```diff
@@ -0,0 +1,15 @@
+package com.epam.training.tasks.third.data;
+
+public class CubeDataValidator {
+
+    public boolean validate (double[] data) {
+
+        double sideLength = data[3];
+
+        return data.length == 4 && sideLength > 0;
```

Comment:
magic number (should be a constant), this validation doesn't bring much value, validate String line by pattern